styles.css

```diff
@@ -447,3 +446,280 @@ input::placeholder {
 textarea::placeholder {
   color: #b3bac5;
 }
+
+/* To ignore for moblie */
+
+.desktop-header {
+  display: none;
+}
+
+.card-img1 {
+  display: none;
+}
+
+div.card-info1 {
+  display: none;
+}
+
+div.card-info2 {
+  display: none;
+}
+
+/* Desktop version */
+@media only screen and (min-width: 768px) {
+  .head-container {
+    margin: 0 155px;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+  }
+
+  .menu {
+    display: none;
+  }
+
+  .desktop-header {
+    list-style: none;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    padding: 10px 0 10px 10px;
+    gap: 32px;
+  }
+
+  .nav {
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    color: #344563;
+    text-decoration: none;
+  }
```

Comment:
Your nav bar looks great. As represented by the design, kindly make the nav bar to be fixed while the user scrolls. A look at the representation below in the design shows that's what's expected.

Figma design:
![image](https://user-images.githubusercontent.com/20554583/199761419-bfdf1df3-f762-4523-8c19-2094503e36e1.png)